### PR TITLE
updated readme minor grammatical fixes

### PR DESCRIPTION
**Pull request by Cloud887**
A few minor grammar fixes one slightly more important one. 
1. are => is and is => are (plural, versus singular) 
2. space added by ,but in scrolling section
3. the the (extra the removed)

Thank you for reading.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ If you are completely new to coding, consider completing <a href="https://www.co
 
 ##### 2. Ensure your laptop up to the task of coding *(time for this can vary widely depending upon the age and condition of your laptop)*
 
-  - Is virus and malware free
+  - Are virus and malware free
```
Comment (geluso) [inline README.md]: Keep this as "is." It's singular. "Ensure your laptop is virus and malware free."

Comment (geluso) [reply README.md]: "Ensure your laptop is up to the task of coding"

Comment (geluso) [reply README.md]: How about, "Eliminate all viruses and malware" ?

Comment (Cloud887) [reply README.md]: I added a commit with the suggestion, I could not find the missing "is"

Comment (Cloud887) [reply README.md]: Nevermind I think I found it

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ If you are completely new to coding, consider completing <a href="https://www.co
 
 ##### 2. Ensure your laptop up to the task of coding *(time for this can vary widely depending upon the age and condition of your laptop)*
```
Comment (geluso) [inline README.md]: One more change, can you add "is" here to update the other error we noticed:

"Ensure your laptop is up to the task of coding"

Comment (Cloud887) [reply README.md]: I was certain I added that in the last commit?

**Comment by Cloud887** [comment]
Resolved merge conflict from the update on master with URL changes to ".io"

**Pull request merged**